File: code_templates/actions/php/CreateMediaEntry.php
$entry = new KalturaMediaEntry();
$entry->name = <%- Lucy.code.variable('answers.name') %>;
$entry->mediaType = <%- Lucy.code.variable('answers.mediaType') %>;
$entry = $client->media->add($entry);
$mediaResource = new KalturaUploadedFileTokenResource();
$mediaResource->token = <%- Lucy.code.variable('answers.uploadTokenId') %>;

$result = $client->media->addContent($entry->id, $mediaResource);
while ($result->status != 2) {
  sleep(1);
  $result = $client->media->get($result->id);
}
<%- Lucy.returnCode('$result') %>

File: code_templates/actions/ruby/CreateMediaEntry.rb
entry = Kaltura::KalturaMediaEntry.new
entry.name = <%- Lucy.code.variable('answers.name') %>
entry.media_type = <%- Lucy.code.variable('answers.mediaType') %>
token = <%- Lucy.code.variable('answers.upload_token_id') %>
entry = @@client.media_service.add_from_uploaded_file(entry, token)

while entry.status != "2" do
  sleep 1
  entry = @@client.media_service.get(entry.id)
end
<%- Lucy.returnCode('entry') %>
